src/temgymbasic/components.py

```diff
@@ -210,10 +210,16 @@ def gui_wrapper():
 
 class Source(Component):
     def __init__(
-        self, z: float, tilt_yx: Tuple[float, float] = (0., 0.), name: Optional[str] = None
+        self, z: float,
+        wavelength: float,
```

Comment:
I think `wavelength` should be a parameter with default value since it doesn't in itself impact the raytracing.